src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private int radius = 0;
+
+    public void setData() {
+        while (radius < 3) {
+            this.radius = new Random().nextInt(300);
+        }
+    }
+
+    @Override
+    public String getName() {
+        return "circle";
+    }
+
+    @Override
+    public double getArea() {
+        setData();
```

Comment:
we can remove setDate() after you create a constructor with radius initialization